test/accessibility/a11yTest.js

```diff
@@ -30,132 +32,129 @@ const commonSessionData = {
     back: []
 };
 
-let stepNum = 0;
+const warning = ['WCAG2AA.Principle1.Guideline1_3.1_3_1.H48'];
+const excludeWarnings = r => {
+    return !warning.includes(r.code);
+};
```

Comment:
would it be worth adding a comment here just to explain why this warning is being ignored? just to avoid any confusion in future?